chapters/methodology/methodology.tex

```diff
@@ -5,8 +5,10 @@ \chapter{Methodology}
 deep-penetration radiation transport problems. In general, the variance reduction
 parameters generated by CADIS and FW-CADIS are not sufficient for
 problems that are strongly
-anisotropic with respect to the flux. The method will extend existing
-methods to generate importance maps--that in turn generate variance reduction
+anisotropic with respect to the flux. The new method will extend existing
```

Comment:
You changed 
> The method will extend existing methods to generate importance maps--that in turn generate variance reduction parameters... 
to 
> The method will extend existing methods to generate adjoint scalar fluxes--that in turn generate variance reduction parameters ..
with the comment
> Note: an importance map is itself frequently used to do VR without modification, so I think saying adjiont scalar fluxes is more precise.

My response was to address the choice of using importance map vs. adjoint flux.